airflow/providers/databricks/provider.yaml

```diff
@@ -92,6 +92,12 @@ hooks:
     python-modules:
       - airflow.providers.databricks.hooks.databricks_sql
 
+sensors:
+  - integration-name: Databricks SQL
+    python-modules:
+      - airflow.providers.databricks.sensors.databricks_sql
+      - airflow.providers.databricks.sensors.databricks_partition
+      - airflow.providers.databricks.sensors.databricks_table_changes
```

Comment:
Please lets remove the `databricks_` prefix from files names (also in the tests), these files are in the databricks provider there is no need for the prefix (also to be compliant with AIP-21)

Reply:
Yes, refactored the file names.